task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -14,6 +14,7 @@
     ExecutionFramework
 )
 from task_processing.plugins.mesos.translator import mesos_status_to_event
+from task_processing.plugins.smartstack import get_mesos_master
```

Comment:
I think that this is the wrong abstraction for this. The mesos executor should take just a connection ip, and the caller which is paasta has the smartstack information.